lib/ansible/modules/system/sysrc.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019 David Lundgren <dlundgren@syberisle.net>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+author:
+    - David Lundgren (@dlundgren)
+module: sysrc
+short_description: Manage FreeBSD using sysrc
+version_added: '2.10'
+description:
+    - Manages /etc/rc.conf for FreeBSD
+options:
+    name:
+        description:
+            - Name of variable in $dest to manage.
+        type: str
+        required: true
+    value:
+        description:
+            - The value if "present"
+        type: str
+    state:
+        description:
+            - Whether the var should be present or absent in $dest.
+            - append/subtract will add or remove the value from a list.
+        type: str
+        default: "present"
+        choices: [ present, absent, append, subtract ]
```

Comment:
I'd call append as item_present, args_present, or value_present. Likewise for subtract item_absent. To be declarative.

Reply:
I went with `value_present` and `value_absent` since we have `value` as the data